pkgs/development/python-modules/csv2md/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, buildPythonPackage
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "csv2md";
+  version = "1.1.1";
```

Comment:
It seems that 1.2.0 was released 3 months ago.
https://github.com/lzakharov/csv2md/releases/tag/v1.2.0